sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/tests/SessionRecords/DatabaseAccountOperationsTests/DatabaseAccountCRUDTests.json

```diff
@@ -1,28 +1,28 @@
 {
```

Comment:
This test record leaked some credentials, please see the CI for the failure details and fix it, thanks.